SerialPrograms/Source/CommonFramework/ImageTools/WaterfillUtilities.h

```diff
@@ -61,6 +62,15 @@ bool match_template_by_waterfill(
     double rmsd_threshold,
     std::function<bool(Kernels::Waterfill::WaterfillObject& object)> check_matched_object);
 
+// Similar to match_template_by_waterfill(), except instead of matching by template,
+// matches by filters, and returns a vector of ImageFloatBox
+std::vector<ImageFloatBox> match_filters_by_waterfill(

```

Comment:
So this function is identical to `match_template_by_waterfill()`, but without the template check?

Reply:
That's right.
I didn't end up using it in the end, so we can delete if you think it's not useful/redundant.

Reply:
Yeah let's get rid of it for now since it's basically copy-paste of the original. If we need it, we can try to merge them.